### PR TITLE
Project 2: Components

### PR DESCRIPTION
**Pull request by ntzwilly**
## Description: 
- Create a components directory.
- Add a new Calculator.js file inside components.
- Create a React component in Calculator.js
- Develop the UI of the calculator
- Use class-based components.
-  import Calculator and use it in App.js.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```
Comment (henatan99) [inline src/App.js]: - [x] The objective of this milestone is to practice React class components. Kindly refactor the `App` component to a React class component. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @ntzwilly , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [x] Kindly remove the boilerplate files in your public directory (images and icons). :pray: 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (ntzwilly) [reply src/App.js]: Thank you. It is done. The App is now a class component

File: src/index.js [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Calculator from './components/Calculator';
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root'),
-);
```
Comment (GzimAsani) [inline src/index.js]: - [ ] You shouldn't have deleted the App component and created a class with the name of App inside the index.js file. The `index.js` should be present as it was and the App.js file should be present as it was only as a class-based component and not a function-based component.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @ntzwilly           
Good job so far! Your project looks almost done :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good use of gitflow :+1: 

## Required Changes ♻️ 
- [x] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (ntzwilly) [reply src/index.js]: It is done

Comment (henatan99) [reply src/index.js]: Nice job! This issue is well addressed. :clap: :+1: :+1:

**Review by henatan99 — APPROVED** [review]
Hi @ntzwilly , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**